easybuild/easyconfigs/c/CUDA/CUDA-7.5.18-iccifort-2015.3.187-GNU-4.9.3-2.25.eb

```diff
@@ -37,7 +37,6 @@ installopts = '-override compiler'
 host_compilers = ["icpc", "g++"]
 
 # Be careful and have a message consistent with the generated wrappers
-modloadmsg = "nvcc uses g++ as the default host compiler. If you want to use icpc as a host compiler you can use"
-modloadmsg += " nvcc_icpc, or nvcc -ccbin=icpc. Likewise, a g++ wrapper called nvcc_g++ has been also created."
+modloadmsg = "nvcc uses g++ as the default host compiler. If you want to use icpc as a host compiler you can use nvcc_icpc, or nvcc -ccbin=icpc. Likewise, a g++ wrapper called nvcc_g++ has been also created.\n"
```

Comment:
@SethosII same here, this line is too long, please break it up using `modloadmsg +=`